pkgs/servers/mx-puppet-discord/node-composition.nix

```diff
@@ -0,0 +1,17 @@
+# This file has been generated by node2nix 1.9.0. Do not edit!
+
+{pkgs ? import <nixpkgs> {
```

Comment:
That probably needs to be relative.

Reply:
Good spot, this is the same with all the `node-composition.nix` files I checked. I've fixed it here